albinos-editor/src/albinospkg/cli.nim

```diff
@@ -27,6 +29,15 @@ var globalHelp = "Usage:\n" & msgTable["help_cmd_msg"] & msgTable[
     "exit_cmd_msg"] & msgTable["clear_cmd_msg"] & msgTable[
     "config_create_cmd_msg"]
 
+proc completion(word: cstring, lc: ptr Completions) {.cdecl.} =
```

Comment:
I'm pretty sure this is not the right way to do auto-completion, but maybe I'm misunderstanding Nim's library usage.
Otherwise, "he" should obviously complete to "help" and it seems weird to specify a series of ifs manually (and not have an array and iterate on it, checking if the beginning matches with it)

Reply:
I followed this tutorial of the official library: https://github.com/antirez/linenoise#completion

It's a c library so we can discuss about it here.

Reply:
I think the link is pretty clear. Obviously you can test it but currently typing "he<TAB>" should not autocomplete properly.